apps/backend/src/donations/donations.controller.ts

```diff
@@ -21,7 +21,6 @@ export class DonationsController {
 
   @Get('orders')
   filter(@Body() filterDonationsDto: FilterDonationsDto) {
```

Comment:
I think it's better to replace the @Body() with @Query() because we usually do @Query() with GET requests and @Body with POST or PUT
https://dev.to/shameel/nestjs-request-param-body-query-headers-ip-5e6k

Reply:
do you mean instead of using a FilterDonationsDto to put all of the filter conditions in the url to use @Query?

Reply:
fixed